mailchimp_subscriber.py

```diff
@@ -56,9 +60,18 @@ def load_conf(conf_file):
     containing the MC List Id, MC User, and MC API key"""
     config = configparser.ConfigParser()
     config.read(conf_file)
+    send_mc_email = False
+    try:
+        if (config['DEFAULT'].getboolean('SendMCEmail',
+                                         fallback=False)):
+            send_mc_email = True
+    except ValueError:
+        pass
```

Comment:
I see you're using a getter with a default value specified. Does this actually throw an exception?

Reply:
also...does this throw an exception?

Reply:
Nope, it doesn't, I'll get rid of the try/except